README.md

```diff
@@ -7,19 +7,56 @@ theme: jekyll-theme-midnight
 
```

Comment:
I believe this remained before I found a better way to implement the rendering.
Could you remove the `theme` from the header?